glean-core/rlb/src/system.rs

```diff
@@ -73,6 +73,18 @@ pub fn get_os_version() -> String {
     parse_linux_os_string(whatsys::kernel_version().unwrap_or_else(|| "Unknown".to_owned()))
 }
 
+#[cfg(target_os = "windows")]
+/// Returns the Build ID for Windows kernels versions >= 10.0
+pub fn get_windows_build_number() -> String {
+    whatsys::windows::get_windows_build_number().unwrap_or_else(|| "Unknown".to_string())
+}
+
+#[cfg(not(target_os = "windows"))]
+/// Returns the Build ID for Windows kernels versions >= 10.0
+pub fn get_windows_build_number() -> String {
+    "Unknown".to_string()
```

Comment:
hey @badboy I am not sure this is correct. I would expect this to more appropriately be None, but I need the non-windows on here right? However, when I tried to wrap these function as an option, the compiler fought me a bit.

Is my instinct right that this should be Option<String>, and None in the `not(target_os="windows")` case? and I just need to figure out how to make it work?

Reply:
Yup, should be `fn get_windows_build_number() -> Option<String>` in both cases.
Wait, why string? didn't we say we go with a number?